src/test/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/DefaultAccessEgressTest.java

```diff
@@ -0,0 +1,24 @@
+package org.opentripplanner.routing.algorithm.raptoradapter.transit;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.street.search.state.TestStateBuilder;
+
+class DefaultAccessEgressTest {
+
+  @Test
+  void containsDriving() {
+    var state = TestStateBuilder.ofDriving().streetEdge().streetEdge().streetEdge().build();
```

Comment:
Can the `.streetEdge` be moved to `TestStateBuilder`?

Reply:
It is already! https://github.com/opentripplanner/OpenTripPlanner/blob/39d3727be2bc57912fa4d4723ae0801a617bcc12/src/test/java/org/opentripplanner/street/search/state/TestStateBuilder.java#L47